Floof branch/master

TuckTuckFloof opened the pull request:
## Description

I added all of the files from EU ISO, I was wondering why it didn't work and it turns out I was trying to push master lmao. ANYWAY, I can see literally all of my commits from my creative process of EU ISO so I guess I'm here to make sure this all gets pushed onto the main branch! If you don't know what it is, it's basically a planck with some isos slapped on the side, greatest thing I've ever made!

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [x] New feature
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

ridingqwerty commented:
There are some issues with this submission.  I don't have a problem with the personal keymap changes being included with a new board PR (although it's easier and cleaner if such changes are done as separate PRs), but there are changes to some unrelated default board files and some other folks' keymaps such that this probably can't be accepted as-is.

tzarc commented:
At this stage, you're going to need to:

1. Remove all changes unrelated to `keyboards/EU_ISOlation`
2. Rename it to `keyboards/eu_isolation` as per QMK naming standards.

You've also removed the entire set of the PR checkboxes, so at this point I'm inclined to just close this PR and ask you to create a new PR once you've made the modifications as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist). If you feel like recovering the content of the PR description, feel free, but this is likely going nowhere until that's done.

We have a process that needs to be followed.

TuckTuckFloof commented:
Hello, yea, I can do that, I already changed the name of the folder. I'd
say nuke it if that entails what I would assume would be yeeting all the
commits. Thanks for the feedback, this is my first PR here.

On Mon, Oct 12, 2020 at 9:30 PM Nick Brassel <notifications@github.com>
wrote:

> At this stage, you're going to need to:
>
>    1. Remove all changes unrelated to keyboards/EU_ISOlation
>    2. Rename it to keyboards/eu_isolation as per QMK naming standards.
>
> You've also removed the entire set of the PR checkboxes, so at this point
> I'm inclined to just close this PR and ask you to create a new PR once
> you've made the modifications as per the PR checklist
> <https://docs.qmk.fm/#/pr_checklist>. If you feel like recovering the
> content of the PR description, feel free, but this is likely going nowhere
> until that's done.
>
> We have a process that needs to be followed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qmk/qmk_firmware/pull/10635#issuecomment-707479196>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKVVMNBTNLKCJQWH6RCQIYDSKPJYBANCNFSM4SN2ADYQ>
> .
>

drashna commented on keyboards/EU_ISOlation/config.h:
```diff
@@ -0,0 +1,33 @@
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
```suggestion
#define PRODUCT_ID      0x4373
```

From hash generated:
```sh
echo -n "EU ISOlation" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```

drashna commented on keyboards/EU_ISOlation/config.h:
```diff
@@ -0,0 +1,33 @@
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    TuckTuckFloof
+#define PRODUCT         EU ISOlation
+#define DESCRIPTION     A terrible fucking creation involving a quad iso macro pad
```
This is not used, and should be removed. 
```suggestion
```

drashna commented on keyboards/EU_ISOlation/info.json:
```diff
@@ -0,0 +1,174 @@
+    "width": 14,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_EUISO_ORTHO": {
```
```suggestion
        "LAYOUT_euiso_ortho": {
```

drashna commented on keyboards/EU_ISOlation/info.json:
```diff
@@ -0,0 +1,174 @@
+            ]
+        },
+
+        "LAYOUT_EUISO_MIT": {
```
```suggestion
        "LAYOUT_euiso_mit": {
```

drashna commented on keyboards/EU_ISOlation/info.json:
```diff
@@ -0,0 +1,174 @@
+            ]
+        },
+
+        "LAYOUT_EUISO_BIGSPACE": {
```
```suggestion
        "LAYOUT_euiso_bigspace": {
```

drashna commented on keyboards/EU_ISOlation/keymaps/FloofBS/config.h:
```diff
@@ -0,0 +1,2 @@
+#define COMBO_COUNT 5
```
```suggestion
#pragma once

#define COMBO_COUNT 5
```

And could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

drashna commented on keyboards/EU_ISOlation/keymaps/FloofBS/keymap.c:
```diff
@@ -0,0 +1,80 @@
+#include "EU_ISO.h"
```
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15


Also,
```suggestion
#include QMK_KEYBOARD_H
```

drashna commented on keyboards/EU_ISOlation/keymaps/default_BIGSPACE/config.h:
```diff
@@ -0,0 +1,2 @@
+#define COMBO_COUNT 5
```
```suggestion
#pragma once 

#define COMBO_COUNT 5
```

drashna commented on keyboards/EU_ISOlation/keymaps/default_BIGSPACE/keymap.c:
```diff
@@ -0,0 +1,45 @@
+#include "EU_ISO.h"
```
```suggestion
#include QMK_KEYBOARD_H
```

drashna commented on keyboards/EU_ISOlation/keymaps/default_ORTHO/keymap.c:
```diff
@@ -0,0 +1,45 @@
+#include "EU_ISO.h"
```
```suggestion
#include QMK_KEYBOARD_H
```

drashna commented on keyboards/eu_isolation/EU_ISO.h:
```diff
@@ -0,0 +1,44 @@
+
+#define ___ KC_NO
+
+#define LAYOUT_EUISO_MIT( \
```
```suggestion
#define LAYOUT_euiso_mit( \
```

drashna commented on keyboards/eu_isolation/EU_ISO.h:
```diff
@@ -0,0 +1,44 @@
+    { ___, ___, k32, k33, k34, k35, ___, k37, k38, k39, k3a, k3b, k3c, k3d } \
+}
+
+#define LAYOUT_EUISO_ORTHO( \
```
```suggestion
#define LAYOUT_euiso_ortho( \
```

drashna commented on keyboards/eu_isolation/EU_ISO.h:
```diff
@@ -0,0 +1,44 @@
+    { ___, ___, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d } \
+}
+
+#define LAYOUT_EUISO_BIGSPACE( \
```
```suggestion
#define LAYOUT_euiso_bigspace( \
```

drashna commented on keyboards/eu_isolation/config.h:
```diff
@@ -0,0 +1,33 @@
+
```
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

TuckTuckFloof commented on keyboards/40percentclub/gherkin/config.h:
```diff
@@ -15,15 +15,15 @@
-#define MATRIX_ROW_PINS { F7, B1, B3, B2, B6 }
-#define MATRIX_COL_PINS { B4, E6, D7, C6, D4, D0 }
+#define MATRIX_ROW_PINS { C6, D7, E6, B4, B5 }
+#define MATRIX_COL_PINS { B2, B3, B1, F7, F6, F5 }
 #define UNUSED_PINS
 
 /* COL2ROW or ROW2COL */
 #define DIODE_DIRECTION COL2ROW
 
 /* number of backlight levels */
-#define BACKLIGHT_PIN B5
+#define BACKLIGHT_PIN B6
```
That was an attempt to fix a usb-c I put on backwards lol. It didn't work...

drashna commented:
the branch is still intact. but you'd need to checkout `floofBranch/master` instead of just `master`
